hal/src/boron/network/sara_ncp_client.h

```diff
@@ -131,6 +140,7 @@ class SaraNcpClient: public CellularNcpClient {
     int modemSetUartState(bool state) const;
     void waitForPowerOff();
     int getAppFirmwareVersion();
+    int defaultParserConfig();
```

Comment:
For my reference, why `defaultParsesConfig()` only for `sara_ncp_client`?

Reply:
No reason honestly. We can probably do the same thing for quectel_ncp_client as well.